pulp_ansible/app/modelresource.py

```diff
@@ -30,6 +30,9 @@ class CollectionVersionContentResource(BaseContentResource):
     Resource for import/export of ansible_collectionversion-content entities.
     """
 
+    contents = fields.Field(
+        column_name="contents", attribute="contents", widget=JSONWidget(), default=list()
```

Comment:
Nice find :+1:

Reply:
Bonus points if you want to add a comment here too:

```
# required for jsonb fields in PostgreSQL database
```

Reply:
Actually, their introspection handles it well, the problem happens when the default is list.
For the other JSONFields that had dict as default, I didn't need to do anything